### DISCUSSION TITLE
VSCode users, please upgrade to Copilot version 1.43

### DISCUSSION THREAD

**Discussion by davecheney** [community/community#30716] [Copilot Conversations]
VSCode pilots, please upgrade to version 1.43 (or later)

_Note_: If you're running VSCode 1.67 or earlier, you must first upgrade VSCode to unlock newer versions of the Copilot extension.

**Comment by tmlewin** [comment]
> VSCode pilots, please upgrade to version 1.43 (or later)
> 
> _Note_: If you running VSCode 1.67 or earlier, you must first upgrade VSCode to unlock newer versions of the Copilot extension.

@davecheney  what newer features if i may ask? because the changelog is quite dated.

**Reply by davecheney** [reply]
@tmlewin i'm sorry I don't have a changelog for you. In general changes to the client are for bug fixes around integration with the parent editor. Features are delivered on the server side and generally don't need a companion client change.

**Reply by Nuchanncat** [reply]
> @tmlewinฉันขอโทษ ฉันไม่มีบันทึกการเปลี่ยนแปลงสำหรับคุณ โดยทั่วไปการเปลี่ยนแปลงในไคลเอนต์มีไว้สำหรับการแก้ไขจุดบกพร่องเกี่ยวกับการรวมเข้ากับตัวแก้ไขหลัก ฟีเจอร์ต่างๆ จะถูกจัดส่งทางฝั่งเซิร์ฟเวอร์ และโดยทั่วไปแล้วไม่จำเป็นต้องเปลี่ยนไคลเอนต์ที่แสดงร่วม

**Reply by Sagor2214** [reply]
what newer features if i may ask? because the changelog is quite dated

**Reply by Andygoldschmitz** [reply]
What does that strange signgs mean

**Reply by Andygoldschmitz** [reply]
I still do not understand what i have to do to do something good

**Reply by Oooomo** [reply]
مشاركتك ودعمك لمنتجاتنا!

مصنوع من♥ ️بواسطة GitHub

اه اوه! لا يمكنك

**Comment by schwertlowe** [comment]
Hi @davecheney. I upgraded to v 1.43, but it did not help me. I have always the same error. "GitHub Copilot could not connect to server." What can I do for that? Thanks in advance.

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by schwertlowe** [reply]
[INFO] [auth] [2022-08-26T08:15:34.599Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-08-26T08:15:34.603Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-08-26T08:16:08.901Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-08-26T08:16:08.902Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Reply by schwertlowe** [reply]
It is interesting, that Copilot works correctly with IntelliJ

**Reply by davecheney** [reply]
@schwertlowe 
as a technical preview user you received complementary access to copilot until August the 22nd. If you wish to continue to use Copilot, you can sign up for a subscription plan here , https://github.com/github-copilot/signup

**Reply by schwertlowe** [reply]
@davecheney I have already paid on 22. of August to go on using GitHub Copilot for a yearly subscription. That's why, I ask that how is it possible that I couln't use Copilot in VS Code?

**Reply by davecheney** [reply]
Do you have multiple GitHub accounts?

**Reply by davecheney** [reply]
@schwertlowe i've confirmed your github login has a yearly subscription. Is it possible vscode is logged in as a different github user?

**Reply by schwertlowe** [reply]
@davecheney No, I logged in with the same github user. How can I solve that problem?

**Reply by schwertlowe** [reply]
If I click the link in VS Code to log in, my github account opens automatically.

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by schwertlowe** [reply]
@davecheney Thanks a lot Dave. It works :)

**Reply by schwertlowe** [reply]
@davecheney Hi Dave. I have another question. I could not use Copilot on my firm's computer. Is there any Proxy configurations to use Copilot on this computer?

**Reply by tmlewin** [reply]
@davecheney  any eta when copilot  would start offering  support for vpn based connections.

**Reply by schwertlowe** [reply]
OK So we will wait for that

**Reply by davecheney** [reply]
Please consult https://github.com/community/community/discussions/29127

**Comment by soethiha2** [comment]
Thanks

**Reply by 1501575808** [reply]
copilot如何退费啊，自动扣费了，找不到关闭的入口

**Reply by Ayingv** [reply]
> Thanks 

Ok

**Comment by Ayingv** [comment]
> VSCode pilots, please upgrade to version 1.43 (or later)
> 
> _Note_: If you're running VSCode 1.67 or earlier, you must first upgrade VSCode to unlock newer versions of the Copilot extension.

VcCode
